packages/js/src/executors/tsc/lib/normalize-options.ts

```diff
@@ -57,5 +57,6 @@ export function normalizeOptions(
       outputPath,
       options.main.replace(`${projectRoot}/`, '').replace('.ts', '.js')
     ),
+    generatePackageJson: options.generatePackageJson ?? true,
```

Comment:
Do we need this if the schema defaults to true? This would affect programmatic usage only.

Reply:
Yeah, we need it precisely to ensure that the programmatic usage gets the correct default value and doesn't get broken with this new option.